huaweicloud/services/apig/resource_huaweicloud_apig_throttling_policy_associate.go

```diff
@@ -0,0 +1,200 @@
+package apig
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/apigw/dedicated/v2/throttles"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceThrottlingPolicyAssociate() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceThrottlingPolicyAssociateCreate,
+		ReadContext:   resourceThrottlingPolicyAssociateRead,
+		UpdateContext: resourceThrottlingPolicyAssociateUpdate,
+		DeleteContext: resourceThrottlingPolicyAssociateDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceThrottlingPolicyAssociateImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"policy_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"publish_ids": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func buildPolicyAssociateOpts(instanceId, policyId string, publishIds []string) throttles.BindOpts {
+	return throttles.BindOpts{
+		InstanceId: instanceId,
+		ThrottleId: policyId,
+		PublishIds: publishIds,
+	}
+}
+
+func resourceThrottlingPolicyAssociateCreate(ctx context.Context, d *schema.ResourceData,
+	meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	client, err := config.ApigV2Client(config.GetRegion(d))
+	if err != nil {
+		return diag.Errorf("error creating HuaweiCloud APIG v2 client: %s", err)
+	}
+
+	opt := buildPolicyAssociateOpts(d.Get("instance_id").(string), d.Get("policy_id").(string),
+		utils.ExpandToStringListBySet(d.Get("publish_ids").(*schema.Set)))
+	_, err = throttles.Bind(client, opt)
+	if err != nil {
+		return diag.Errorf("error binding policy to the API: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s/%s", d.Get("instance_id").(string), d.Get("policy_id").(string)))
```

Comment:
suggest to use two variables to save instanceID and policyID

Reply:
done.